Add AWS S3 docs (#35)
transloadit/uppy.io: docs/guides/choosing-upload-strategy.md

If your users are planning to mostly upload small files and/or a lot of files, it’s better to use [`@uppy/aws-s3`][].

> [`@uppy/aws-s3-multipart`][] starts to become valuable for larger files (100&nbsp;MB+) as it uploads a single object as a set of parts.

Reviewer: ```suggestion
[`@uppy/aws-s3-multipart`][] is valuable for larger files (100&nbsp;MB+) as it uploads a single object as a set of parts.
```
